orbit/pkg/packaging/linux_shared.go

```diff
@@ -202,6 +202,130 @@ func buildNFPM(opt Options, pkger nfpm.Packager) (string, error) {
 	return filename, nil
 }
 
+func writeInitScript(opt Options, rootPath string) error {
+	systemdRoot := filepath.Join(rootPath, "etc", "init.d")
+	if err := secure.MkdirAll(systemdRoot, constant.DefaultDirMode); err != nil {
+		return fmt.Errorf("create systemd dir: %w", err)
+	}
+	if err := ioutil.WriteFile(
+		filepath.Join(systemdRoot, "orbit"),
+		[]byte(`#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          orbit
+# Required-Start:    $remote_fs $network
+# Required-Stop:     $remote_fs $network
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: orbit osquery control daemon
+# Description:       This is a daemon that controls the orbit osquery
+### END INIT INFO
+
+PATH=/sbin:/usr/sbin:/bin:/usr/bin:/opt/orbit/bin/orbit/linux/stable
+DESC="orbit osquery control daemon"
+NAME=orbit
+DAEMON=/opt/orbit/bin/orbit/linux/stable/orbit
```

Comment:
`stable` string should actually be parameterized (the user can create packages with different release channels, `edge`, etc.)